src/components/Select.js

```diff
@@ -133,6 +139,14 @@ class Select extends React.Component {
     }
   };
 
+  _handleSearch = (e) => {
+    const filteredOptions = this.props.options.filter((item) => {
+      return item.displayValue.toLowerCase().includes(e.target.value)
```

Comment:
There's no support in Internet Explorer for includes - need to make sure this is tested inside of sabotage and moneymap in ie11 to make sure our babel configs are handling this.

Reply:
Good catch. I've gotten use to the poly fill.

Reply:
Thanks, I will test this.

Reply:
If you're having issues testing in IE 11 you can always just switch to the lodash includes function. https://lodash.com/docs/4.17.15#includes

Reply:
I was able to confirm `includes()` is working in ie11 within sabotage.

![includes_ie11](https://user-images.githubusercontent.com/31394784/62229116-3a7b8180-b37c-11e9-9d59-3b593ccda152.gif)

Reply:
I think it is important to see if if works in IE11 within the docs here. It's possible if works in Sabotage because Babel is providing a poly fill there but that it won't work here on it's own. You can run the local dev server in this repo to spin up the docs.

Reply:
@cerinman 
I Updated the component to use lodash `includes` as you recommended.  I verified this works in the docs on ie11.